File: lab/c/soma_todos_os_10/soma_todos_10.c
#include <stdio.h>

int soma_10_linear();

int soma_10_gaus();

int main(){
    int linear, gaus_form;

    linear = soma_10_linear();
    printf("Soma Linear: %d\n", linear);

    gaus_form = soma_10_gaus();
    printf("Soma Fórmula de Gaus: %d\n", gaus_form);

}

int soma_10_linear(){
    int soma;

    for (int i=0; i <= 10; i++){
        soma += i;
    }

    return soma;
}

int soma_10_gaus(){
    int soma = (10 / 2) * (0 + 11);
    return soma;
}

File: lab/python/somas/soma_todos_10.py
def soma_10_linear():
    # Forma Linear
    soma = 0
    for x in range(0, 11):
        soma += x

    return soma


def soma_10_modelo_gaus():
    # Modelo de Gaus para progressoes a1...an
    soma = (11 / 2) * (0 + 10)
    return soma


if __name__ == "__main__":
    print(f"modelo linear: {soma_10_linear()}")
    print(f"modelo de Gaus: {soma_10_modelo_gaus()}")
